Core/HLE/sceNetAdhoc.cpp

```diff
@@ -2605,130 +2605,119 @@ static int sceNetAdhocPtpOpen(const char *srcmac, int sport, const char *dstmac,
-				// 	// Generate Port Number
-				// 	sport = (uint16_t)_getRandomNumber(65535);
-				// }
+				//sport 0 should be shifted back to 0 when using offset Phantasy Star Portable 2 use this
+				sport = -(int)portOffset;
 			}
 			
-			// Valid Ports
-			if (!isPTPPortInUse(sport) /*&& dport != 0*/) {
-				//sport 0 should be shifted back to 0 when using offset Phantasy Star Portable 2 use this
-				if (sport == 0) sport = -(int)portOffset;
-				// Valid Arguments
-				if (bufsize > 0 && rexmt_int > 0 && rexmt_cnt > 0) {
-					// Create Infrastructure Socket
-					int tcpsocket = socket(AF_INET, SOCK_STREAM, IPPROTO_TCP);
-					
-					// Valid Socket produced
-					if (tcpsocket > 0) {
-						// Change socket buffer size when necessary
-						if (getSockBufferSize(tcpsocket, SO_SNDBUF) < bufsize) setSockBufferSize(tcpsocket, SO_SNDBUF, bufsize);
-						if (getSockBufferSize(tcpsocket, SO_RCVBUF) < bufsize) setSockBufferSize(tcpsocket, SO_RCVBUF, bufsize);
+			// Valid Arguments
+			if (bufsize > 0 && rexmt_int > 0 && rexmt_cnt > 0) {
+				// Create Infrastructure Socket
+				int tcpsocket = socket(AF_INET, SOCK_STREAM, IPPROTO_TCP);
+
+				// Valid Socket produced
+				if (tcpsocket > 0) {
+					// Change socket buffer size when necessary
+					if (getSockBufferSize(tcpsocket, SO_SNDBUF) < bufsize) setSockBufferSize(tcpsocket, SO_SNDBUF, bufsize);
+					if (getSockBufferSize(tcpsocket, SO_RCVBUF) < bufsize) setSockBufferSize(tcpsocket, SO_RCVBUF, bufsize);
 
-						// Enable KeepAlive
-						setSockKeepAlive(tcpsocket, true, rexmt_int / 1000000L, rexmt_cnt);
+					// Enable KeepAlive
+					setSockKeepAlive(tcpsocket, true, rexmt_int / 1000000L, rexmt_cnt);
 
-						// Ignore SIGPIPE when supported (ie. BSD/MacOS)
-						setSockNoSIGPIPE(tcpsocket, 1);
+					// Ignore SIGPIPE when supported (ie. BSD/MacOS)
+					setSockNoSIGPIPE(tcpsocket, 1);
 
-						// Enable Port Re-use
-						setsockopt(tcpsocket, SOL_SOCKET, SO_REUSEADDR, (const char*)&one, sizeof(one));
+					// Enable Port Re-use
+					setsockopt(tcpsocket, SOL_SOCKET, SO_REUSEADDR, (const char*)&one, sizeof(one));
 
-						// Apply Default Send Timeout Settings to Socket
-						setSockTimeout(tcpsocket, SO_SNDTIMEO, rexmt_int);
+					// Apply Default Send Timeout Settings to Socket
+					setSockTimeout(tcpsocket, SO_SNDTIMEO, rexmt_int);
 
-						// Disable Nagle Algo to send immediately. Or may be we shouldn't disable Nagle since there is PtpFlush function?
-						if (g_Config.bTCPNoDelay) setSockNoDelay(tcpsocket, 1);
-						
-						// Binding Information for local Port
-						sockaddr_in addr;
-						// addr.sin_len = sizeof(addr);
-						addr.sin_family = AF_INET;
-						addr.sin_addr.s_addr = INADDR_ANY;
-						if (isLocalServer) {
-							getLocalIp(&addr);
+					// Disable Nagle Algo to send immediately. Or may be we shouldn't disable Nagle since there is PtpFlush function?
+					if (g_Config.bTCPNoDelay) setSockNoDelay(tcpsocket, 1);
```

Comment:
Just thought of a coding style thing - when you're changing or adding code, could you put the statement after an if on its own line? I think that reads a lot better. 

Often it can even be nice to put that single line in brackets - makes it easier to add more code later (though for things where that's not likely, omitting the brackets is ok too).

Don't have to fix this now in this PR, but in the future.

Reply:
Yes, i usually put it on next line when writing new code since it's easier to put breakpoints that way, but if it was an old code... i leave it as is LOL

Reply:
Ah didn't realize that line was just moved :)